en/asset/label-atlas.md

```diff
@@ -0,0 +1,32 @@
+# LabelAtlas Asset
+
+**LabelAtlas** asset is a user defined asset, it's used for configuring LabelAtlas.
```

Comment:
**LabelAtlas** asset is a user defined asset, it's used for configuring LabelAtlas.
-->
**LabelAtlas** asset is a user-defined asset, it's used for configuring a **LabelAtlas**.